network/loadbalancers/k8smaster/variables.tf

```diff
@@ -6,6 +6,10 @@ variable "label_prefix" {
   default = ""
 }
 
+variable "master_lb_enabled" {
+  default = true
+}
+
```

Comment:
I think we should stick with the convention of using the string values "true" and "false" for booleans. This would also be inline with your doc updates.

The version of terraform I am using at least is requiring me to set `master_lb_enabled = 1` or `master_lb_enabled = 0` as opposed to `master_lb_enabled = "true"` or `master_lb_enabled = "false"` to avoid a parsing trying to convert "false" to an integer in the `count`.

Reply:
`master_lb_enabled` is defined as a boolean type (same as `etcd_iscsi_volume_create`), so you can set master_lb_enabled = `true` | `false` instead of 1 or 0.

Reply:
`etcd_lb_enabled` is defined as a `string`, in fact, it's better to change to a `boolean` also.
See https://www.terraform.io/docs/configuration/variables.html#booleans

Reply:
From the link"

> It is recommended for now to specify boolean values for variables as the strings "true" and "false", to avoid some caveats in the conversion process. A future version of Terraform will properly support boolean values and so relying on the current behavior could result in backwards-incompatibilities at that time.

Here's why:

```
grep master_oci_lb_enabled terraform.tfvars
master_oci_lb_enabled = "false"
```

```
terraform plan

Error: Error running plan: 2 error(s) occurred:

* module.k8smaster-public-lb.oci_load_balancer.lb-k8smaster: cannot parse "false" as an integer
* module.etcd-lb.oci_load_balancer.lb-etcd: cannot parse "false" as an integer
```
* module.etcd-lb.oci_load_balancer.lb-etcd: cannot parse "false" as an integer
* module.k8smaster-public-lb.oci_load_balancer.lb-k8smaster: cannot parse "false" as an integer